src/preprocess/preprocess_utils.py

```diff
@@ -0,0 +1,48 @@
+import xarray as xr
```

Comment:
This file can be renamed to `utils.py`, since its already in the preprocessing folder